lib/imgix/path.rb

```diff
@@ -12,16 +12,14 @@ def initialize(prefix, secure_url_token, path = "/")
       @secure_url_token = secure_url_token
       @path = path
       @options = {}
-
-      @path = CGI.escape(@path) if /^https?/ =~ @path
-      @path = "/#{@path}" if @path[0] != "/"
     end
 
     def to_url(opts = {})
+      sanitized_path ||= sanitize_path(@path)
```

Comment:
Is the `||=` needed for `sanitised_path`, since it's not an ivar?

```suggestion
      sanitized_path = sanitize_path(@path)
```

Reply:
Nope. Thanks, this snuck through.